apps/api/e2e/tests/admin.test.ts

```diff
@@ -1308,19 +1309,23 @@ describe("Admin API", () => {
     expect(removeResponse.agent.id).toBe(agent1.id);
 
     // Verify agent1 is no longer in active leaderboard
-    const updatedLeaderboard =
-      (await adminClient.getCompetitionLeaderboard()) as LeaderboardResponse;
-    expect(updatedLeaderboard.success).toBe(true);
-    const activeAgentIds = updatedLeaderboard.leaderboard.map(
-      (entry) => entry.agentId,
+    const updatedLeaderboard = await adminClient.getCompetitionAgents(
+      competition.id,
+      { sort: "rank", includeInactive: true },
     );
+    expect(updatedLeaderboard.success).toBe(true);
+    if (!updatedLeaderboard.success) throw new Error("Failed to get agents");
```

Comment:
remove the thrown error and use `expect`

Reply:
yeah, it actually already has the `expect`, but without this there's a build error cause the compiler thinks the type could be an error.  Anyway I changed it and added type casting instead, debatable which is cleaner though